xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -284,15 +275,9 @@ void CGUIDialogPVRTimerSettings::InitializeSettings()
   if (tag->SupportsFolders())
     AddEdit(group, SETTING_TMR_DIR, 19076, 0, tag->m_strDirectory, true, false, 19104);
 
-  AddToggle(group, SETTING_TMR_RADIO, 19077, 0, tag->m_bIsRadio);
```

Comment:
if possible we should move skin related strings into their own language file/add-on. That doesn't mean that skins might not reuse core labels, but it would allow skins to share more common labels.